contracts/implementations/Mapper.sol

```diff
@@ -18,4 +47,21 @@ contract Mapper {
             sourceNode
         ];
     }
+
+    /// @notice Gets the beneficiary associated with the pair idProxy/nodeId
+    /// @param idProxyAddress The address of the NFT proxy
+    /// @param nodeId The node Id to be queried
+    function getBeneficiary(address idProxyAddress, uint256 nodeId)
+        external
+        view
+        returns (address beneficiary)
+    {
+        beneficiary = MapperStorage.getStorage().beneficiaries[idProxyAddress][
+            nodeId
+        ];
+
+        if (beneficiary == address(0)) {
```

Comment:
A note about this default in the doc would be helpful to me. Do you think it's good behavior? I think so.

Reply:
I think makes sense in the business logic and does not hurt having it here